include/swift/Frontend/ModuleInterfaceLoader.h

```diff
@@ -248,6 +262,9 @@ class ExplicitModuleMapParser {
         result.moduleDocPath = val.str();
       } else if (key == "sourceInfoPath") {
         result.moduleSourceInfoPath = val.str();
+      } else if (key == "isFramework") {
+        std::istringstream is(val.str());
+        is >> std::boolalpha >> result.isFramework;
```

Comment:
Please check for `true`/`false` specifically here rather than using the standard library's I/O streams.